src/main/java/hw3/HomePage.java

```diff
@@ -0,0 +1,74 @@
+package hw3;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class HomePage extends AbstractPage {
+
+    private static final String URL = "https://epam.github.io/JDI/";
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+
+    @FindBy(id = "name")
+    private WebElement usernameTextField;
+
+    @FindBy(id = "password")
+    private WebElement passwordTextField;
+
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(id = "user-name")
+    private WebElement usernameLabel;
```

Comment:
These elements could be moved to the `AbstractPage` or separate `Page Component`